notary-signer/Dockerfile

```diff
@@ -10,11 +10,11 @@ WORKDIR ${INSTALLDIR}
 
 RUN set -eux; \
     apk add --no-cache --virtual build-deps git go make musl-dev; \
-    go version | grep 'go1.10.8 '; \
+    go version | grep 'go1.12.[[:digit:]]* '; \
```

Comment:
The point of this check was to check an exact version, so we know which version it is built with, so please switch to 1.12.8

Reply:
Gotcha; was doing this so that go1.12.9 wouldn't make the build fail, but later on wasn't sure if that was correct. I'll update